r2dbc-spi/src/main/java/io/r2dbc/spi/Statement.java

```diff
@@ -165,4 +165,14 @@ default Statement bind(int index, short value) {
      */
     Publisher<? extends Result> execute();
 
+    /**
+     * Configures {@link Statement} to return the generated values from any rows created by this {@link Statement} in the {@link Result} returned from {@link #execute()}.  If no columns are specified,
+     * implementations are free to choose which columns will be returned.  If called multiple times, only the columns requested in the final invocation will be returned.
```

Comment:
Nit: There seem to be two spaces between the dots and each `If`.

Reply:
I think this is true for all the R2DBC documentation 😄.  I'm a double-spacer, especially in fixed-width text where kerning can't help sentence breaks.